integration_tests/src/main/python/date_time_test.py

```diff
@@ -262,6 +262,13 @@ def test_unsupported_fallback_to_unix_timestamp(data_gen):
         "to_unix_timestamp(a, b)"),
         "ToUnixTimestamp")
 
+# TODO: has another test for this called test_unix_timestamp
+@pytest.mark.parametrize('data_gen', date_n_time_gens, ids=idfn)
```

Comment:
We should also test string gen.
Please refer to the Product plan doc, it requires String data type.